### DISCUSSION TITLE
Multiple shortcuts for one function

### DISCUSSION THREAD

**Discussion by M0rtenB** [mikekazakov/nimble-commander#295] [Ideas]
I'm wishing for a way to have multiple keyboard shortcuts correspond to the same function.
In casu: "Go to enclosing folder" available by backspace, left key and cmd-up (Finder style).

I know this was asked in https://magnumbytes.com/forum/viewtopic.php?t=455 but just throwing it in here that this would be a lovely addition to a wonderful piece of software.

**Comment by mikekazakov** [comment]
Fully agree, it'll be a nice addition. Though I don't known when/if I'll be able to work on that, so any help is welcome.
